app.py

```diff
@@ -154,14 +161,17 @@ def plot_vector(df):
 
     aim_callback = AimCallbackHandler(
         repo=".",
```

Comment:
We should make this location configurable through an env var. Will make deployment easier if we decide to share it as a volume with an aimstack sidecar

Reply:
agreed